modules/mediawiki/files/bin/deploy-mediawiki.py

```diff
@@ -33,12 +33,34 @@ def run(args, start):
         os.system(f'/usr/local/bin/logsalmsg {text}')
     else:
         print(text)
+    if args.world:
+        if args.pull:
+            pull = str(args.pull).split(',')
+            if 'world' not in pull:
+                pull.append('world')
+                args.pull = ','.join(pull)
+            pull = None
+        else:
+            args.pull = 'world'
+    if args.pull:
+        pull = str(args.pull).split(',')
+        if 'config' in pull:
+            os.chdir('/srv/mediawiki-staging/config')
+            exitcodes.append(os.system('sudo -u www-data git pull --recurse-submodules --quiet'))
```

Comment:
no submodules in this repo but if we're gonna use same command then stick it in a variable and don't repeat

Reply:
Will do, thanks!